package.json

```diff
@@ -182,12 +182,12 @@
         "category": "ONE"
       },
       {
-        "command": "onevscode.circle-tracer",
+        "command": "one.viewer.circleTracer",
         "title": "circle tracer",
         "category": "ONE"
       },
       {
-        "command": "onevscode.circle-graphview",
+        "command": "one.viewer.circleNetron",
```

Comment:
FYI, this graph viewer literally runs netron-circle.

Reply:
Hmm... `circle-graphview` invokes `CircleGraphPanel`. I think it's good to remain as it is. Or if the code writer agrees with fixing, it would be okay.

Reply:
it's not `circleNetron`. purpose of this view is to show circle graph view, using netron codes, for development of partition editor, and maybe other features.
UI is not same as circle netron tool, so renaming to `circleNetron` is incorrect.

anyway this feature is for UI development and I am planning to remove this command someday.

Reply:
@YongseopKim @seanshpark  Sure.  Will one.viewer.circleGraph do?